packages/components/psammead-section-divider/src/index.test.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
+import { latin, arabic } from '@bbc/gel-foundations/scripts';
+import SectionDivider from './index';
+
+describe('SectionDivider', () => {
+  describe('With title', () => {
+    shouldMatchSnapshot(
```

Comment:
This tests that `ltr` is the default, can we have one that passes it in explicitly please :)